src/test/java/org.graylog.integrations/AWSTestingUtils.java

```diff
@@ -0,0 +1,60 @@
+package org.graylog.integrations;
```

Comment:
It looks like this file resides in the `src/test/java/org.graylog.integrations/` directory. Usually this is a nested directory structure. (e.g. `src/test/javaorg/graylog/integrations/`)

(same for all other test files in this PR)

Is this on purpose?

Reply:
Thanks for catching this. Fixed in https://github.com/Graylog2/graylog-plugin-integrations/pull/116/commits/1d8c3156128f30ccca992a85a1f8fe7b768c8ae6